### PR TITLE
Second PR -  simrin_joshi.md

### PR DESCRIPTION
**Pull request by simrin051**
I am a passionate professional focused on full stack web development.
I have 1 year of professional experience in Angular and i am rigorously looking for opportunities in Node , Express and Database to mold me to become rich experienced full stack development.

### CODE REVIEW

**Comment by proful** [comment]
Pls review  @twindle-co/developer**

**Comment by Akshay2996** [comment]
@simrinjoshi the markdown file is empty. Please see it.

**Comment by PuruVJ** [comment]
The markdown is empty, but I don't see any issue in it. She can add herself to team.md later

File: docs/TEAM.md [hunk 1]
```diff
@@ -72,8 +72,13 @@
...
- |
+| <img src="./images/team/aditya.jpg" width="80px">                                                                                                                                                                                                              | [Aditya Satpute](https://github.com/Foxtrot-1418)                     | I'm a Full Stack Web Developer From Nashik, Maharashtra, India. I've experience in HTML, CSS, JS, Jquery, Bootstrap, PHP and currently working fo r Multinational Company as AWS Developer.                                                                                                                                                                |
+ | <img src="./images/team/simrin_joshi.jpg" 
+ width="80px">
```
Comment (RafaelBatman55) [inline docs/TEAM.md]: Keep the  code in the same line.

image

File: docs/TEAM.md [hunk 2]
```diff
@@ -72,8 +72,13 @@
...
- |
+| <img src="./images/team/aditya.jpg" width="80px">                                                                                                                                                                                                              | [Aditya Satpute](https://github.com/Foxtrot-1418)                     | I'm a Full Stack Web Developer From Nashik, Maharashtra, India. I've experience in HTML, CSS, JS, Jquery, Bootstrap, PHP and currently working fo r Multinational Company as AWS Developer.                                                                                                                                                                |
+ | <img src="./images/team/simrin_joshi.jpg" 
+ width="80px">      
+         
+        | [Simrin](https://github.com/simrin051) | 1.5 yr of experience from Bangalore India | I am a 
+passionate professional focussed on full stack web development. I have 1 year of professional experience in Angular , HTML ,CSS ,JavaScipt, Java and i am rigorously looking for opportunities in Node , Express and Database to enrich me in  full stack developement.  I want to actively contribute to Twindle.
```
Comment (RafaelBatman55) [inline docs/TEAM.md]: Remove this space between "I am a" and "passionate professional..."

image

These kind of space splits your text between two blocks.

**Review by RafaelBatman55 — CHANGES_REQUESTED** [review]
Check these issues to avoid to break lines.

**Review by PuruVJ — APPROVED** [review]

**Pull request closed without merging**